### ISSUE TITLE
[Screen Reader]: screen reader does not read values sent to LED display

### ISSUE THREAD

**Issue by thomasjball** [microsoft/pxt-microbit#7061]
The scenario here is simple. The values (numbers, strings, icons) send to the LED display are not read out by the screen reader, even when focus is on the simulator.

1. Consider the following program:
```
input.onButtonPressed(Button.A, function() {
    basic.showString("Hello!")
})
```

2. Turn on your favorite screen reader.

3. Press the A button in the microbit simulator. See that the micro:bit display scrolls the string "Hello!"

4. Note that the screen reader does nothing.
